addons/account/views/report_payment_receipt_templates.xml

```diff
@@ -43,9 +43,10 @@
                         </tr>
                     </thead>
                     <tbody>
-                        <t t-foreach="o.move_id._get_reconciled_invoices_partials()" t-as="rec">
+                        <t t-set="partials" t-value="o.move_id._get_reconciled_invoices_partials()"/>
+                        <t t-set="invoices" t-value="{partial[2].move_id.id: partial[2].move_id for partial in partials}.values()"/>
```

Comment:
`{partial[2].move_id for partial in partials}`

Reply:
Hello, thanks for the review!

The idea of putting it into a dic is that it avoids to have twice the invoice. It's a bit hacky but if I do it the way you do, you will have the two invoices with the same id in the list.

But while I'm saying that, I might check another way to do it. That is, see if it is not possible to modify `_get_reconciled_invoices_partials()` instead

Reply:
my suggestion is to put the invoice in a `set`, so you will have the same end result but it is more readable as you don't have to write a dict comprehension to just extract the values

Reply:
Nevermind, it works with `{partial[2].move_id for partial in partials}` ^^